### PR TITLE
Update samples to use containerd

### PR DESCRIPTION
**Pull request by shannonxtreme**
Related issue https://github.com/kubernetes/website/issues/30976

The output samples in this page used docker as the runtime. I've switched to samples taken from a cluster running containerd.

Deploy preview here: https://deploy-preview-31781--kubernetes-io-main-staging.netlify.app/docs/tasks/debug-application-cluster/debug-application-introspection/

/sig docs
/language en

### CODE REVIEW

File: content/en/docs/tasks/debug-application-cluster/debug-application-introspection.md [hunk 1]
```diff
@@ -231,42 +242,88 @@ spec:
...
-  hostIP: 10.240.0.9
+        startedAt: "2022-02-17T21:51:05Z"
+  hostIP: 192.168.0.113
   phase: Running
-  podIP: 10.244.0.6
-  startTime: 2016-03-24T01:39:49Z
+  podIP: 10.88.0.3
+  podIPs:
+  - ip: 10.88.0.3
+  - ip: 2001:4860:4860::3
```
Comment (sftim) [inline content/en/docs/tasks/debug-application-cluster/debug-application-introspection.md:324]: Please use a [documentation IP address](https://www.rfc-editor.org/rfc/rfc3849.txt) here.

File: content/en/docs/tasks/debug-application-cluster/debug-application-introspection.md [hunk 2]
```diff
@@ -40,70 +40,77 @@ kubectl get pods
...
+Node:         kube-worker-1/192.168.0.113
+Start Time:   Thu, 17 Feb 2022 16:51:01 -0500
+Labels:       app=nginx
+              pod-template-hash=67d4bdd6f5
+Annotations:  <none>
+Status:       Running
+IP:           10.88.0.3
+IPs:
+  IP:           10.88.0.3
+  IP:           2001:4860:4860::3
```
Comment (sftim) [inline content/en/docs/tasks/debug-application-cluster/debug-application-introspection.md:67]: Please use a [documentation IP address](https://www.rfc-editor.org/rfc/rfc3849.txt) here.

**Review by sftim — CHANGES_REQUESTED** [review]
One thing to fix, otherwise LGTM.

**Comment by chrisnegus** [comment]
It looks like the IPv6 IP address has been fixed. If that's it, can someone merge this?
/lgtm

**Pull request merged**